edx_solutions_api_integration/courses/views.py

```diff
@@ -1,96 +1,70 @@
-    get_course_child_key,
-)
-from edx_solutions_api_integration.models import (
-    CourseGroupRelationship,
-    CourseContentGroupRelationship,
-    GroupProfile,
-)
-from progress.models import CourseModuleCompletion
-from social_engagement.models import StudentSocialEngagementScore
-from edx_solutions_api_integration.permissions import SecureAPIView, SecureListAPIView, MobileAPIView
-from edx_solutions_api_integration.users.serializers import UserSerializer, UserCountByCitySerializer
-from edx_solutions_api_integration.utils import (
-    generate_base_uri,
-    str2bool,
-    get_time_series_data,
-    parse_datetime,
-    get_ids_from_list_param,
-    strip_xblock_wrapper_div,
-    get_cached_data,
-    cache_course_data,
-    cache_course_user_data,
-    get_aggregate_exclusion_user_ids,
-    get_user_from_request_params,
-    css_param_to_list,
-)
-from edx_solutions_api_integration.courses.serializers import (
-    CourseSerializer,
-    GradeSerializer,
-    CourseCompletionsLeadersSerializer,
-    CourseSocialLeadersSerializer,
-    CourseProficiencyLeadersSerializer,
-)
-from progress.serializers import CourseModuleCompletionSerializer
+from xmodule.modulestore.search import path_to_location
 
+from edx_solutions_api_integration.courses.serializers import (BlockCompletionSerializer,
+                                                               CourseCompletionsLeadersSerializer,
+                                                               CourseProficiencyLeadersSerializer, CourseSerializer,
+                                                               CourseSocialLeadersSerializer, GradeSerializer,
+                                                               UserGradebookSerializer)
+from edx_solutions_api_integration.courses.utils import (generate_leaderboard, get_num_users_started,
+                                                         get_total_completions, get_user_position)
+from edx_solutions_api_integration.courseware_access import (course_exists, get_course, get_course_child,
+                                                             get_course_child_key, get_course_key)
+from edx_solutions_api_integration.models import (CourseContentGroupRelationship, CourseGroupRelationship, GroupProfile)
+from edx_solutions_api_integration.permissions import MobileAPIView, SecureAPIView, SecureListAPIView
+from edx_solutions_api_integration.users.serializers import UserCountByCitySerializer, UserSerializer
+from edx_solutions_api_integration.utils import (cache_course_data, cache_course_user_data, css_param_to_list,
+                                                 generate_base_uri, get_aggregate_exclusion_user_ids, get_cached_data,
+                                                 get_ids_from_list_param, get_time_series_data,
+                                                 get_user_from_request_params, parse_datetime, str2bool,
+                                                 strip_xblock_wrapper_div)
```

Comment:
Should probably keep the previous formatting style: One import per line, starting on the line below, indented four spaces.